### PR TITLE
docs(layouts): more layout fixes

### PR DESCRIPTION
**Pull request by layershifter**
Continues work from #1014.

### CODE REVIEW

File: docs/src/components/ComponentDoc/ExampleSection.tsx [hunk 1]
```diff
@@ -14,19 +14,15 @@ export type ExampleSectionProps = Extendable<{
...
-  </Flex>
+const ExampleSection: React.FC<ExampleSectionProps> = ({ title, children }) => (
+  <>
+    <Header as="h2" styles={headerStyle} className="no-anchor">
+      {title}
+    </Header>
+    <Grid variables={{ gridGap: '2rem' }} columns="1">
+      {children}
+    </Grid>
+  </>
```
Comment (layershifter) [inline docs/src/components/ComponentDoc/ExampleSection.tsx:25]: I don't see any reason to use there `Flex`...

File: docs/src/components/ComponentDoc/ComponentExample/ComponentExample.tsx [hunk 2]
```diff
@@ -79,7 +78,7 @@ class ComponentExample extends React.Component<ComponentExampleProps, ComponentE
       knobs: this.getDefaultKnobsValue(),
       showCode: this.isActiveHash(),
       componentVariables: {},
-      showRtl: examplePath && examplePath.endsWith('rtl') ? true : false,
+      showRtl: examplePath && examplePath.endsWith('rtl'),
```
Comment (layershifter) [inline docs/src/components/ComponentDoc/ComponentExample/ComponentExample.tsx:81]: Simplified condition

File: docs/src/components/ComponentDoc/ComponentExample/ComponentExample.tsx [hunk 3]
```diff
@@ -56,7 +56,6 @@ interface ComponentExampleState {
 const childrenStyle: React.CSSProperties = {
   paddingTop: 0,
   paddingBottom: '10px',
-  maxWidth: pxToRem(500),
```
Comment (layershifter) [inline docs/src/components/ComponentDoc/ComponentExample/ComponentExample.tsx:59]: Fixes this issue:
image
https://stardust-ui.github.io/react/components/animation#types-keyframe-params

File: docs/src/components/ComponentDoc/ComponentDoc.tsx [hunk 4]
```diff
@@ -137,28 +137,23 @@ class ComponentDoc extends React.Component<any, any> {
...
-            </Flex.Item>
-
-            <Flex.Item>
-              <div style={exampleEndStyle}>
-                This is the bottom <Icon name="pointing down" />
-              </div>
-            </Flex.Item>
-
-            {/* TODO: bring back the right floating menu
+        <Grid columns="auto 300px" styles={{ justifyContent: 'normal', justifyItems: 'stretch' }}>
```
Comment (layershifter) [inline docs/src/components/ComponentDoc/ComponentDoc.tsx:140]: Fixes this issue:

image
https://stardust-ui.github.io/react/components/form

File: docs/src/examples/components/Text/States/TextExampleTruncated.tsx [hunk 5]
```diff
@@ -11,7 +11,7 @@ const [notTruncatedText, truncatedText] = [
 )
 
 const TextExampleTruncated = () => (
-  <div style={{ overflow: 'hidden', textOverflow: 'ellipsis' }}>
+  <div style={{ overflow: 'hidden', textOverflow: 'ellipsis', width: 500 }}>
```
Comment (layershifter) [inline docs/src/examples/components/Text/States/TextExampleTruncated.tsx:14]: Fixes: https://stardust-ui.github.io/react/components/text#states-truncated

image

**Review by mnajdova — APPROVED** [review]
Changes looks good to me! Are we planning to move the dropdown for switching teams back in the `Sidebar` or at least put some absolute positioning in order to prevent scrolling on each changing?

**Comment by layershifter** [comment]
> Are we planning to move the dropdown for switching teams back in the Sidebar

As I know there is a plan to move back theme selector

**Pull request merged**